website/source/docs/config/index.html.md

```diff
@@ -111,6 +111,22 @@ For Consul, the following options are supported:
 
   * `token` (optional) - An access token to use to write data to Consul.
 
+  * `tls_ca_file` (optional, mandatory when scheme is https) - The path to the CA certificate
```

Comment:
These are not mandatory. If no `ca_file` is provided the system bundle (default) should be used. If `cert` and `key` are not provided, then mutual TLS is not done, but we can verify the server authenticity.